apps/preview/nuxt/pages/showcases/Modal/ModalTransition.vue

```diff
@@ -51,7 +51,13 @@
 </template>
 
 <script lang="ts" setup>
+import { ref, watch } from 'vue';
 import { SfModal, SfButton, SfIconClose, useDisclosure } from '@storefront-ui/vue';
-
+import { useFocus } from '@vueuse/core';
+const buttonRef = ref();
+const { focused } = useFocus(buttonRef);
```

Comment:
Do you have to explicitly set and take away focus from the button while Focus Trap will move focus to the first tabbable item in the trap? I don't think so. There's always only one element focused at a time, so if the trap intercepts active element, the button will lose focus anyway.

Note [activeElement](https://developer.mozilla.org/en-US/docs/Web/API/Document/activeElement)

Reply:
@dkacper  So I need to set focus only when closing modal? Is there a simpler way to do that?

Reply:
OK, I see! You use `useFocus` to move focus back to the element that triggered the modal, rather than moving it away to modal after it opens. It makes sense as long as we do not restore focus in the Focus Trap. I think we could add this functionality to the Focus Trap (it'd be as easy as saving `activeElement` on activate, and restoring focus on that element on deactivate).